### ISSUE TITLE
UX: Notifications show ID rather than Name

### ISSUE THREAD

**Issue by ekerstein** [OpenBazaar/openbazaar-desktop#936]
Someone just followed me. When I clicked notifications I saw their ID, not their name. The ID isn't useful information, whereas the name of the user would be. Showing the name of the user in the notifications would improve user experience.

![screen shot 2017-09-23 at 3 58 52 pm](https://user-images.githubusercontent.com/6055508/30776747-f6d9ff9a-a079-11e7-8dd7-cdc3bc6f73e9.png)

**Comment by rmisio** [comment]
This is an ongoing UX discussion. In the vast majority of places (there are many) across the client, we should the users handle (if they have one), otherwise we fall back to the peerID.

But then... there's a third identifier with the store name coming into play.

Seems super confusing to me. My first question would be... do we really need both a handle and store name (could those somehow be collapsed into one)?

In the end, whatever we decide we could show, it's just a matter of making a decision and ideally being consistent across the app (seems weird that in most places we show the handle and then in some place (I want to say Order overlay) we show the store name).

**Comment by morebrownies** [comment]
Yeah so we've run into a bit of an unexpected issue with handles. We were expecting to have handles fully integrated into OpenBazaar for the 2.0 launch, but unfortunately handles got delayed, so for now we only have peerID and store name to work with.

That has left us in an awkard position because the peerID is the only unique way to identify a user/store. 

Store names are not unique and have the potential of being spoofed pretty easily and that can lead to users' sending money to fake accounts.

With that said, not many people are going to know what the proper peerID is for a store either. So I agree with @ekerstein that we need to consider displaying `Store name` in all places. This has been brought up by a few other people as well.

> But then... there's a third identifier with the store name coming into play.

Since we don't have handles yet, and it's not entirely sure when we will have them, I think we should proceed with using store name in all places. Once we have time to spend a sprint or two on handles, we can then also figure out how to properly layer them into the UI.

> Seems super confusing to me. My first question would be... do we really need both a handle and store name (could those somehow be collapsed into one)?

Yeah, I think we need both. Most social networks have a similar pattern of a `name` identifier and a `handle` identifier. See Twitter for an example. Name can be whatever you'd like (non-unique) and a handle is unique. 

Thoughts?
